README.md

```diff
@@ -1 +1,236 @@
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+> This is Leaderboard Project. We used _HTML, CSS & JAVASCRIPT_ to create this Leaderboard Project.
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> - [Live Demo Link]
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> I Used Gitflow, modules, and advanced JS (ES6) in the Leaderboard Project.
+
+<details>
+  <summary>Clien</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+     <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+     <li><a href="https://www.w3schools.com/css/">Javascript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Html structure with Button, Inputs, etc]**
+- **[Styling css with colors, borders, paddings, margins & sizes, etc]**
+_ **[javascript functions, modules, ES6 syntax, variables, etc]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> I will use API to store user data in the Leaderboard Project.
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: to Clone the repo.
+<!--
+Example command:
+_git clone https://github.com/Atril33/Leaderboard.git
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+_ Opend CMD, Terminal or GitBash in your pc and type this
+ _git clone command _https://github.com/Atril33/Leaderboard.gitt_
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+https://github.com/Atril33/Todo_List_ES6.git
+```
+--->
+
+### Install
+
+Install this project with:
+_ You need to run this command _npm i_
+Then run this comman _npm run dev_
+<!--
+Example command:
+
+```sh
+  cd my-project
+  _git clone https://github.com/Atril33/Leaderboard.git_
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+_ You can use it by download in your computer.
+ This is this link for this project _https://github.com/Atril33/Leaderboard.git_
+<!--
+Example command:
+
+```sh
+  rails server
+### Deployment
+
+You can deploy this project using:
+Not Yet
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors">Syed</a>
+
+> This Project is created by Syed Atril
```

Comment:
- Please, let's keep our readme consistent by removing this as you have a section for an author already.